<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsUI</class>
 <widget class="QMainWindow" name="SettingsUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>163</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="saveButton">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Path</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="pathLineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QCheckBox" name="fetchImagesCheckBox">
      <property name="text">
       <string>Fetch images</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="noCacheCheckBox">
      <property name="text">
       <string>Disable cache</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Theme:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="themeComboBox"/>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
